.github/workflows/ci.yml

```diff
@@ -78,7 +78,7 @@ jobs:
       if: matrix.os == 'macOS-latest'
       run: |
         brew update
-        brew install ace boost eigen swig qt5 orocos-kdl catch2 qhull ipopt cppad pkg-config pybind11 libmatio
+        brew install ace boost eigen swig qt5 orocos-kdl catch2 qhull ipopt cppad pkg-config pybind11 libmatio opencv pcl
```

Comment:
What would be needed for the realsense part?

Reply:
Ah yes! I totally forgot. For the realsense part, atleast in Linux we might need `librealsense2` and `libralsense2-dev`. However we might have to add the relevant servers to get the repositories, I suppose.

I need to understand requirements for windows and macOS.

Reply:
At least for compilation, `librealsense` is a package on both vcpkg and homebrew.

Reply:
The repo should be given in https://github.com/IntelRealSense/librealsense/blob/master/doc/distribution_linux.md#installing-the-packages .

Reply:
Modified the CI in https://github.com/dic-iit/bipedal-locomotion-framework/pull/165/commits/b0503562515a360b2c2c3b0c0e47205545cfbc34.